README.md

```diff
@@ -153,10 +154,22 @@ Kilobars              | kilobars, kbar
 Pounds per Square Inch| psi
 Atmosphere            | atmosphere, at
 
+### Volume
+
+Unit                  | Names
+----------------------|---------------------------------------
+Litres                | litres, liters, L, l
+Millilitres           | millilitres, milliliters, ml, mL
+Gallons               | gallons, gal
+Quart                 | quart, qt
```

Comment:
We should probably be more consistent about plural vs. non-plural names, but this seems ok for now. Ditto for pint.